README.md

```diff
@@ -24,6 +24,38 @@
 * Mapstruct
 * Lombok
 
+## Start the application using docker
+* To run the app you need to add .env file to the root of the project. Please ask administrator for the file.
+
+* download the app
+```
+git clone https://github.com/KSM-development/wst-backoffice.git
+```
+* go to main app directory
+```
+cd wst-backoffice
+```
+
+* create a jar file
+```
+mvn clean package
```

Comment:
Please change to mvn clean package -Dmaven.test.skip=true

Reply:
done